rrfs-test/testinput_expr/namelist.atmosphere

```diff
@@ -0,0 +1,56 @@
+&nhyd_model
```

Comment:
@SamuelDegelia-NOAA  That will be very helpful if a "doc " block could be given on those set ups.  where have they come from ?  If they should be used as a starting point for our rrfs tests/experiments even outside of rdasapp.

Reply:
The `namelist.atmosphere` and `streams.atmosphere` here are just copies of what was previously in `rrfs-test/testinput`. I moved that directory into `rrfs-test/testinput_expr` since I believe some people were using those files for other experiments. 

As for where they came from, these files were added in #91 when we added the new MPAS test case. Would @guoqing-noaa, @Junjun-NOAA, or @chunhuazhou be able to respond to this? It might be better to do this in another PR, too.

Reply:
@SamuelDegelia-NOAA  So , you meant you were using the set up from GSL 's case,  that is informative..  That will be good if you add those information into this namelist file.   In the future, we  need further discussion on those setup , we know where to start from.

Reply:
This file is copied from rrfs-workflow. The MPAS model user guide should have detailed description about each namelist parameters and most of those setting are decided by MODEL or Physics guys.

Reply:
Here is the v8.0.1 user guide:
https://www2.mmm.ucar.edu/projects/mpas/mpas_atmosphere_users_guide_8.0.1.pdf

They may have a more recent user guide for v8.2.2